systemtest/src/test/java/io/strimzi/systemtest/Resources.java

```diff
@@ -294,6 +308,46 @@ private DoneableKafkaConnectS2I kafkaConnectS2I(KafkaConnectS2I kafkaConnectS2I)
         });
     }
 
+    DoneableKafkaMirrorMaker kafkaMirrorMaker(String name, String sourceBootstrapServer, String targetBootstrapServer, String groupId, int mirrorMakerReplicas, boolean tlsListener) {
+        return kafkaMirrorMaker(defaultMirrorMaker(name, sourceBootstrapServer, targetBootstrapServer, groupId, mirrorMakerReplicas, tlsListener).build());
+    }
+
+    private KafkaMirrorMakerBuilder defaultMirrorMaker(String name, String sourceBootstrapServer, String targetBootstrapServer, String groupId, int mirrorMakerReplicas, boolean tlsListener) {
+        return new KafkaMirrorMakerBuilder()
+            .withMetadata(new ObjectMetaBuilder().withName(name).withNamespace(client().getNamespace()).build())
+            .withNewSpec()
+                .withNewConsumer()
+                    .withBootstrapServers(tlsListener ? sourceBootstrapServer + "-kafka-bootstrap:9093" : sourceBootstrapServer + "-kafka-bootstrap:9092")
+                    .withGroupId(groupId)
+                .endConsumer()
+                .withNewProducer()
+                    .withBootstrapServers(tlsListener ? targetBootstrapServer + "-kafka-bootstrap:9093" : targetBootstrapServer + "-kafka-bootstrap:9092")
+                .endProducer()
+            .withReplicas(mirrorMakerReplicas)
+            .withWhitelist(".*")
+            .endSpec();
+    }
+
+    private DoneableKafkaMirrorMaker kafkaMirrorMaker(KafkaMirrorMaker kafkaMirrorMaker) {
+        return new DoneableKafkaMirrorMaker(kafkaMirrorMaker, k -> {
+            TestUtils.waitFor("Kafka Mirror Maker creation", 60000, 5000,
```

Comment:
Maybe we should remove magic numbers which appear multiple time in the code and replace them with constants ?